docs/api/props.md

```diff
@@ -0,0 +1,20 @@
+## Props
+
+propName | type | required | default | description
+---------|------|----------|---------|------------
+className|`string`|no|''|Custom className
+dropAreaClassName|`string`|no|''|Custom className for the DropArea
+fileInfoClassName|`string`|no|''|Custom className for the FileInfo
+imageEditorClassName|`string`|no|''|Custom className for the ImageEditor
+dragOnDocument|`boolean`|no|true|Listen for drag events in the whole document
+dropOnDocument|`boolean`|no|false|Allow to drop on document
+label|`string`|yes||Label for the input
+metadataComponent|`React Component`|no|null|Custom component for the metadata. Props: `name`, `size`, `extension`, `type`
+thumbnailComponent|`React Component`|no|null|Custom component for the image thumbnail. Props: `children` (`<img>` node with the thumbnail)
+displayImageThumbnail|boolean|no|true|Whether to generate a thumbnail for image files
+cropAspectRatio|`number`|no|0|If cropTool is enabled, the aspect ratio for the selection. 0 means the selection is free
+cropTool|boolean|no|false|Whether to render a crop tool for image files
+scaleOptions| ``` {width: number, height: number: keepAspectRatio: boolean}```| no|null|Scale option for file images. `keepAspectRatio` refers to if the original aspect ratio should be kept when applying scaling
```

Comment:
would it be possible to break this JSON into many lines in markdown?

I see in the rendered document that it's too wide to fit in one line so it's broken into a few lines so it would b good if each line contained exactly one field

Reply:
I deleted the table and I made the page style the same as it is in react-carousel docs